QUARKUS-1159.md

```diff
@@ -0,0 +1,37 @@
+# QUARKUS-1159 - Quarkus FIPS support over RHEL8
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1159
+
+The goal is ensure Quarkus runs on a FIPS enabled RHEL 8 System
+
+## Scope of testing
+
+### Impact on test suites and testing automation
+- Test will be implemented in https://github.com/quarkus-qe/quarkus-test-suite (Openshift scenarios are out of scope) 
+  
+### Impact on resources
+
+FIPS coverage will have its own instances with FIPS enabled and will cover JDK 11 and 17. The impact is pretty big because we will run the current Quarkus test suite twice, in JVM and native mode. The best approach is implement this coverage with a Jenkins matrix job in order to 
```

Comment:
```suggestion
FIPS coverage will have use Jenkins runner instances with enabled FIPS support and will be executed using JDK 11 and 17 and equivalent Mandrel images. The impact is pretty big because we will run the some test suites twice, in both JVM and native mode. The best approach is implement this coverage with a Jenkins matrix job in order to 
```